bucket/bulk-crap-uninstaller.json

```diff
@@ -1,10 +1,14 @@
 {
+    "version": "4.16",
+    "description": "Bulk program uninstaller with advanced automation",
     "homepage": "https://www.bcuninstaller.com/",
-    "description": "Bulk program uninstaller with advanced automation.",
     "license": "Apache-2.0",
-    "version": "4.16",
-    "url": "https://www.fosshub.com/Bulk-Crap-Uninstaller.html/BCUninstaller_4.16_portable.zip",
+    "url": "https://github.com/Klocman/Bulk-Crap-Uninstaller/releases/download/v4.16/BCUninstaller_4.16_portable.zip",
     "hash": "d383996cc87a82bbb6c51f775d711c887db6e44687d848b6fb0792fde703d548",
+    "pre_install": "Copy-Item \"$persist_dir\\BCUninstaller.settings\" \"$dir\" -ErrorAction SilentlyContinue -Force",
+    "uninstaller": {
+        "script": "Copy-Item \"$dir\\BCUninstaller.settings\" \"$persist_dir\" -ErrorAction SilentlyContinue -Force"
```

Comment:
This does not work. It will only copy `BCUninstaller.settings` to a `bulk-crap-uninstaller` file when there is no `bulk-crap-uninstaller` directory.